addons/website/static/src/js/backend/view_hierarchy/view_hierarchy.js

```diff
@@ -0,0 +1,250 @@
+/** @odoo-module **/
+
+import { HierarchyNavbar } from "./hierarchy_navbar";
+import { Layout } from "@web/search/layout";
+import { registry } from "@web/core/registry";
+import { useService } from "@web/core/utils/hooks";
+import { Component, onWillStart, useEffect, useState} from "@odoo/owl";
+
+export class ViewHierarchy extends Component {
+    setup() {
+        this.action = useService("action");
+        this.orm = useService("orm");
+        this.router = useService("router");
+        this.state = useState({ showInactive: false, searchedView: {}, viewTree: {} });
+        this.websites = useState({ names: new Set(["All Websites"]), selected: "All Websites" });
+        this.viewId = this.props.action.context.active_id || this.router.current.hash.active_id;
+        this.hideGenericViewByWebsite = {};
+
+        onWillStart(async () => {
+            const { sibling_views, hierarchy } = await this.orm.call(
```

Comment:
I think that we don't have this rule in our linter. If I remember correctly, having a variable with underscore is fine if it comes from the backend.
I will put an alias to the destructuring to avoid your linter error 😄